docs/docsite/rst/roadmap/ROADMAP_2_12.rst

```diff
@@ -0,0 +1,43 @@
+.. _core_roadmap_2_12:
+
+=================
+Ansible-core 2.12
+=================
+
+.. contents::
+   :local:
+
+Release Schedule
+----------------
+
+Expected
+========
+
+PRs must be raised well in advance of the dates below to have a chance of being included in this ansible-core release.
+
+.. note:: There is no Alpha phase in 2.12.
+.. note:: Dates subject to change.
+
+- 2021-08-30 Feature Freeze
+  No new functionality (including modules/plugins) to any code
+
+- 2021-09-13 Beta 1
+- 2021-09-20 Beta 2 (if necessary)
+
+- 2021-10-04 Release Candidate 1 (and ``stable-2.12`` branching from ``devel``)
+- 2021-10-11 Release Candidate 2 (if necessary)
+
+- 2021-10-25 Release
+
+Release Manager
+---------------
+
+ Ansible Core Team
+
+Planned work
+============
+
+- Bump the minimum Python version requirement for the controller to Python 3.8. This will be a hard requirement.
+- Introduce split-controller testing in ``ansible-test`` to separate dependencies for the controller from dependencies on the target.
```

Comment:
I would have prefered to hear that ansible-test is slowly moving outside core, so it can be used as a standalone development tool, with its own release cycle.